src/index.html

```diff
@@ -1,13 +1,81 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css"
+      integrity="sha512-NmLkDIU1C/C88wi324HBc+S2kLhi08PN5GDeUVVVC/BVt/9Izdsc9SVeVfA1UZbY3sHUlDSyRXhCzHfr6hmPPw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    >
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__img">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__img card__img--imac"
+        >
+      </div>
+
+      <div class="card__name">
+        <h2 class="card__description">
+           APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+      </div>
+
+      <div class="card__code">
+        Product code: 195434
+      </div>
+
+      <div class="card__conteiner">
+        <div class="rate">
+          <div class="rate__stars rate__stars--4">
+            <div class="rate__star"></div>
+
+            <div class="rate__star"></div>
+
+            <div class="rate__star"></div>
+
+            <div class="rate__star"></div>
+
+            <div class="rate__star"></div>
+          </div>
+
+          <div class="rate__text">
+            Reviews: 5
+          </div>
+        </div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__priceText">
+           Price:
+        </div>
+
+        <div class="card__value">
+          $2,199
+        </div>
+      </div>
+
+      <div class="card__buy">
+        <a
+          href="#"
+          class="card__button"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </div>
```

Comment:
Don't create additional wrappers to existing elements, you can move all styles to card__button class